docs/source/markdown/podman-pod-create.1.md

```diff
@@ -41,7 +41,15 @@ Examples of the List Format:
 #### **--device**=_host-device_[**:**_container-device_][**:**_permissions_]
 
 Add a host device to the pod. Optional *permissions* parameter
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge error.

Reply:
Thanks, I was confused why validation was failing